examples/e-commerce/src/app.mobile.css

```diff
@@ -0,0 +1,261 @@
+    opacity: 1;
+    transition-delay: 0;
+  }
+
+  .container-filters .container-header,
+  .container-filters .container-body {
+    padding: 2rem 2rem 0 2rem;
+  }
+
+  .filtering .header-logo {
+    left: 50%;
+    position: absolute;
+    top: 1.5rem;
+    transform: translateX(-50%);
+  }
+
+  .filtering .header-title {
+    display: none;
+  }
+
+  .filtering .container-filters {
+    position: absolute;
+    transform: translateY(4rem);
+  }
+
+  .filtering .container-results {
+    display: none;
+  }
+
+  .container-filters-footer {
+    background-color: #fff;
+    border-top: 1px solid #ebecf3;
+    bottom: 4rem;
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+    position: sticky;
+    width: 100%;
+    z-index: 1;
+  }
+
+  .container-filters-footer-button-wrapper {
+    width: calc(50% - 0.5rem);
+  }
+
+  .ais-ClearRefinements-button,
+  .container-filters-footer .button {
+    background-color: rgba(65, 66, 71, 0.08);
+    border: none;
+    border-radius: 8px;
+    cursor: pointer;
+    display: flex;
+    font: inherit;
+    font-size: 0.75rem;
+    font-weight: bold;
+    justify-content: center;
+    padding: 12px;
+    text-align: center;
+    width: 100%;
+  }
+
+  .container-filters-footer .button-primary {
+    background-color: #e2a400;
+    color: #fff;
+  }
+
+  /* Filters button that triggers the overlay */
+
+  .filtering .filters-button {
+    display: none;
+  }
+
+  .filters-button {
+    align-items: center;
+    background-color: #e2a400;
+    border: none;
+    border-radius: 8px;
+    bottom: 2rem;
+    box-shadow: 0 4px 22px 0 rgba(185, 135, 0, 0.5);
+    color: #fff;
+    cursor: pointer;
+    display: flex;
+    font: inherit;
+    font-size: 0.875rem;
+    font-weight: bold;
+    justify-content: center;
+    left: 50%;
+    min-height: 40px;
+    min-width: 112px;
+    position: fixed;
+    transform: translateX(-50%);
+  }
+
+  .filters-button svg {
+    height: 14px;
+    margin-right: 8px;
+    width: 16px;
+  }
+
+  .container {
+    padding: 4rem 2rem;
+  }
+
+  .container-options {
+    display: none;
+  }
+
+  .header {
+    background: linear-gradient(to bottom, #ae8600, #885b01);
+    background-image: url('./images/cover-mobile.png');
+    background-image: url('./images/cover-mobile.png'),
+      linear-gradient(to bottom, #ae8600, #885b01);
+    background-position: bottom;
+    background-size: cover;
+    min-height: 300px;
+    position: relative;
+    transition: transform 200ms ease-out;
+  }
+
+  /* SearchBox */
+
+  .header .ais-SearchBox {
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    transform: translateY(50%);
+    width: 100vw;
+  }
+
+  .header .ais-SearchBox .ais-SearchBox-form {
+    margin: auto;
+    max-width: 90%;
+  }
+
+  /* RefinementList */
+
+  .ais-RefinementList-list {
+    display: grid;
+    grid-auto-flow: column;
+    grid-template-rows: repeat(5, 1fr);
+  }
+
+  .ais-RefinementList-item {
+    flex: 50%;
```

Comment:
![image](https://user-images.githubusercontent.com/2982512/59934499-f7a8be80-944b-11e9-91bf-bf17691bdd3f.png)

Reply:
Fixed in c7622d086cc465125468d9e94797c0dee69b739b.